### PR TITLE
Added option to override the default selectors

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

**Comment by aschempp** [comment]
Any questions on the PR, or could this get merged? 😊

**Comment by AniDubEA** [comment]
> 
>

**Comment by AniDubEA** [comment]
> 
>

**Review by AniDubEA — APPROVED** [review]

File: assets/anchor-style.js [hunk 1]
```diff
@@ -3,5 +3,5 @@ gitbook.events.bind('start', function(e, config) {
 });
 
 gitbook.events.bind('page.change', function() {
-    anchors.add('h1,h2,h3,h4,h5');
+    anchors.add(anchors.options.selector || 'h1,h2,h3,h4,h5');
```
Comment (AniDubEA) [inline assets/anchor-style.js:6]:
```suggestion
    anchors.add(anchors.options.selector || 'h1,h2,h3,h4,h5');
```

**Comment by Simran-B** [comment]
This was implemented in https://github.com/Simran-B/gitbook-plugin-anchorjs/commit/cfb4f06abff5cba19c34b92f7f929137dc098cec#diff-b27a85c1747ac2f404851b90ae2531ffacbd23f8289bf63ef88e356ae3c09396

**Pull request closed without merging**